build.sh

```diff
@@ -62,6 +80,11 @@ for PKG_PATH in $(catkin_topological_order --only-folders); do
   test -f "$PKG_PATH/COLCON_IGNORE" && echo "Skipped" && continue
   (
   cd "$PKG_PATH"
+  GENERATED_DEBIAN_PACKAGE=ros-${ROS_DEB}$(catkin_topological_order --only-names | tr '_' '-')
+  if [ -n "$CONTINUE_PACKAGE_GENERATION" ] && [ -n "$GENERATED_DEBIAN_PACKAGE" ] && ls "/home/runner/apt_repo/${GENERATED_DEBIAN_PACKAGE}_"*.deb >/dev/null 2>&1; then
```

Comment:
the variable and the extra check seems overkill.

Reply:
Removed